src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/model/person/OtherIdentifiers.kt

```diff
@@ -1,11 +1,6 @@
 package uk.gov.justice.digital.hmpps.personrecord.model.person
 
-import uk.gov.justice.digital.hmpps.personrecord.model.identifiers.CROIdentifier
-import uk.gov.justice.digital.hmpps.personrecord.model.identifiers.PNCIdentifier
-
 data class OtherIdentifiers(
```

Comment:
I know this hasn't been added here but I am finding it hard to understand what the point of this class is. Can these not just be fields on `Person`?
Again not necessary now but something to think about for a future change

Reply:
Yeah i think this data class pre-dates me, definitely one to sort out